scripts/create-admin.py

```diff
@@ -10,13 +10,15 @@
         sys.exit(1)
 
     org_id, email = sys.argv[1:]  # pylint: disable=unbalanced-tuple-unpacking
-    u = User(id=str(uuid.uuid4()), email=email, external_id=email)
-    audit_admin = AuditAdministration(user_id=u.id, organization_id=org_id)
-    db_session.add(u)
+    user = User.query.filter_by(email=email).first()
+    if not user:
+        user = User(id=str(uuid.uuid4()), email=email, external_id=email)
+    audit_admin = AuditAdministration(user_id=user.id, organization_id=org_id)
+    db_session.add(user)
```

Comment:
should this be inside the if?

Reply:
yes, ill move it, though it works fine this way as well